### ISSUE TITLE
Broken Update URL Gives Unhelpful Errors

### ISSUE THREAD

**Issue by riq-dfaubion** [Digital-Sapphire/PyUpdater#122]
When the Update URL is not properly set, or if you run the application without uploading your updates to the update store, when the application is run, all the downloader does it complain about the update size being too large:
```
WARNI [pyupdater.client.downloader:211][MainThread] Downloaded file is very large, reading it in to memory may crash the app
WARNI [pyupdater.client.downloader:211][MainThread] Downloaded file is very large, reading it in to memory may crash the app
```

**Suggestion:**
Differentiate between failed update queries and large update file.  Give appropriate error messages for each.

**Comment by riq-dfaubion** [comment]
Error codes from AWS do not have a content-length in their headers.  I'd suggest that in the case of 404 and 403, `_create_response` in downloader.py should fail over to the next download url.

**Comment by JMSwag** [comment]
@riq-dfaubion Please create a PR with you suggestions & I'll merge. Thanks for looking into this.

**Comment by JMSwag** [comment]
Fixed in 5f3dcbb9ddd4b67709fe199b4d0bf6f674736f93
